.gitignore

```diff
@@ -75,3 +75,7 @@ benchmarks/images
 .amltmp
 *.amltmp 
 *.amltemp
+
+# lifetimes Testing Artifacts
+betageo_fitter.pkl
+exploded.csv
```

Comment:
Do we maybe want to add this into a `test/fixtures` directory so that we can run the test in a CI/CD ?

Reply:
This was copy/pasted straight over from the BG/NBD Time-Invariant Covariates PR in the first repo: https://github.com/CamDavidsonPilon/lifetimes/pull/342

In general I don't want to make any changes to the `fitters` module (or by extension, the tests for that module) because the new Bayesian `models` module will be replacing it, but I'll make an exception here because these external testing artifacts are disrupting test coverage reports.

I'll be merging this PR as-is because I have limited time this weekend before my vacation, but I'll create issues for this and anything else you encounter.